### PR TITLE
feat: added qa automation yml

### PR DESCRIPTION
**Pull request by wannacfuture**
Added QA automation YML
closes https://github.com/ubiquibot/QA-Improvements/issues/1

### CODE REVIEW

**Review by pavlovcik — COMMENTED** [review]
Maybe this should only be on the `development` branch?

You should also associate this with the correct issue if possible.

**Comment by wannacfuture** [comment]
>Maybe this should only be on the development branch?
Correct. 
>You should also associate this with the https://github.com/ubiquibot/QA-Improvements/issues/1 if possible.
You mean, adding issue instead of PR?
@Draeieg  said PR also works for him. I wonder if its okay for you if I try that after checking this pr works.

File: .github/workflows/qa-automation.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Add pull requests to project
+
+on:
+  pull_request:
+    types:
+      - review_requested
```
Comment (pavlovcik) [inline .github/workflows/qa-automation.yml:6]: You need to specify the branch, probably `development`

**Review by pavlovcik — CHANGES_REQUESTED** [review]

File: .github/workflows/qa-automation.yml (hunk 1 above)
Comment (wannacfuture) [reply .github/workflows/qa-automation.yml:6]: You mean, only for the prs whose base is 'development'?

Comment (wannacfuture) [reply .github/workflows/qa-automation.yml:6]: fixed.

**Pull request closed without merging**